mirroring/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:latest
+
+RUN mkdir /mirroring
+WORKDIR /mirroring
+
+COPY ./mirror_log.go ./go.mod ./go.sum ./
+
+RUN go mod download
+
+RUN GO111MODULE=on GOOS=linux go build -o mirroring .
+
+from ubuntu:latest
+WORKDIR /root/
+COPY --from=0 /mirroring/mirroring .
+RUN apt-get -y update && apt-get -y upgrade ca-certificates
+CMD ["./mirroring"]
```

Comment:
Curious, if you use alpine/nix instead of ubuntu, would it work without having this RUN command? (it just happens to make the container non-reproducible)

Reply:
I was able to get the Dockerfile to work with Alpine w/o the RUN command I just ended up having to change the compile flags. It looks like this:
 
```
FROM golang:latest

RUN mkdir /mirroring
WORKDIR /mirroring

COPY ./mirror_log.go ./go.mod ./go.sum ./

RUN go mod download

RUN GO111MODULE=on GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o mirroring .

from alpine:latest
WORKDIR ./
COPY --from=0 /mirroring/mirroring ./mirroring
ENTRYPOINT ["./mirroring"]
```

Should we make the switch to this version of the Dockerfile?

Reply:
LGTM! I susepct the image is also smaller.

Reply:
Yup! Nearly 5 times smaller